### PR TITLE
feat(angular/tooltip): allow zero padding styling

### LINKED ISSUE #939: Feature request: Tooltip without padding/margin
To support our use cases for the tooltip (see screenshot) it'd be good if the paddings/margins could (optionally) be deactivated.

image

### PR DESCRIPTION
**Pull request by jeripeierSBB**
Closes #939

### CODE REVIEW

File: src/angular/tooltip/tooltip.md [hunk 1]
```diff
@@ -53,6 +53,15 @@ To add a delay before showing or hiding the tooltip, you can use the inputs `hov
 To completely disable a tooltip, set `disabled` on `sbb-tooltip` or `sbbTooltipDisabled` on
 `sbbTooltip`. While disabled, a tooltip will never be shown.
 
+#### Remove padding
```
Comment (jeripeierSBB) [inline src/angular/tooltip/tooltip.md:56]:
```suggestion
#### Zero panel padding
```

**Comment by jeripeierSBB** [comment]
Changes to tooltipPanelClass input won't be reflected to an already opened tooltip. I think this should be enough in this case.

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Review by mhaertwig — APPROVED** [review]
Nice, LGTM 👍

**Pull request merged**